kpet/data.py

```diff
@@ -553,6 +554,20 @@ def get_sets_from_regex_list(regex_list):
                                   "in suite: {}\ncase: {}".
                                   format(suite.name, case.name))
 
+    def validate_test_names(self):
+        """
+        Check test names are unique.
+        """
+        name_list = []
+        for suite in self.suites:
+            for case in suite.cases:
+                name_list.append(tuple(filter(lambda x: x is not None,
```

Comment:
This isn't something I'd do, the semantics look a bit complicated. But I'm fine with leaving as-is, if the tests pass.

Reply:
Yeah, this is non-trivial and kind of an overkill for the two-value case. However, we'll have to support arbitrary number of values soonish and this approach will be more appropriate, so I'd rather keep it as is.
Speaking of tests, I don't actually have any, and this is an important feature, so I'd go and write some.
Thanks :)!